backend/codesirius/problems/models/reference_solution.py

```
@@ -0,0 +1,22 @@
+
+
+class ReferenceSolution(BaseSubmission):
+    """
+    Model representing a reference solution.
+    """
+
+    class Meta:
+        verbose_name = "Reference Solution"
+        verbose_name_plural = "Reference Solutions"
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)